pages/user/edit/[userID].js

```diff
@@ -0,0 +1,123 @@
+import { useState, useEffect } from 'react';
+import { useRouter } from 'next/router';
+import Image from 'next/image';
+
+import { Loader } from '../../../components/Loader';
+import { Layout } from '../../../components/Layout';
+import { getUserDetail } from '../../../firebase/api/users';
+import { FormInput } from '../../../components/Form/FormInput';
+import { FormTextarea } from '../../../components/Form/FormTextarea';
+import imgPlaceholder from '../../../public/img/img_placeholder_user.webp';
+import { editUserProfile } from '../../../firebase/api/users';
+import { PrivateRouter } from '../../../HOC/PrivateRouter';
+import { useAuth } from '../../../contexts/AuthContext';
+
+const EditUser = () => {
+  const router = useRouter();
+  const { currentUser } = useAuth();
+
+  const [user, setUser] = useState();
+
+  const [profilePicState, setProfilePicState] = useState('');
+  // review
+
+  useEffect(() => {
+    const handleFetching = async () => {
+      setUser(await getUserDetail(currentUser.uid));
+    };
+
+    if (currentUser.uid) {
+      handleFetching();
+    }
+  }, [currentUser]);
+
+  if (!user) {
+    return <Loader />;
+  }
+
+  const { name, surname, username, bio, profilePic } = user;
+
+  const handleImageChange = (e) => {
+    setProfilePicState(e.target.value);
+  };
+
+  const formSubmit = async (e) => {
+    e.preventDefault();
+
+    const formData = {
+      uid: currentUser.uid,
+      name: e.target.name.value,
+      surname: e.target.surname.value,
```

Comment:
It might be better to stick to proper English. The American one xD. So firstName, lastName. Much clearer in my opinion.

Reply:
![Snímek obrazovky 2022-05-28 v 12 09 04](https://user-images.githubusercontent.com/65444975/170820815-b22c47ac-3530-45f3-b5c6-bfb9b6fe16f0.png)